akka-actor/src/main/scala/akka/dispatch/Mailbox.scala

```diff
@@ -258,7 +258,7 @@ private[akka] abstract class Mailbox(val messageQueue: MessageQueue)
    * Process the messages in the mailbox
    */
   @tailrec private final def processMailbox(
-      left: Int = java.lang.Math.max(dispatcher.throughput, 1),
```

Comment:
I think the Java one is used to avoid boxing. But the Scala one is also something like two invokevirtual and two getstatic while the Java is one invokestatic. This is performance sensitive code since it is in the mailbox processing logic. Better leave it as it is.

Reply:
@johanandren np, yah performance matters